includes/admin-templates/new-template.php

```diff
@@ -47,14 +49,6 @@
 			);
 			?></div>
 		<div id="elementor-new-template__description__content"><?php echo esc_html__( 'Use templates to create the different pieces of your site, and reuse them with one click whenever needed.', 'elementor' ); ?></div>
-		<?php
```

Comment:
What is this?

Reply:
old code - it was commented out  so i cleaned the file (boy scout rule )